src/actions-on-google.ts

```diff
@@ -280,6 +284,26 @@ export class ActionsOnGoogle {
     /** @public */
     deviceInstanceId = 'default'
 
+    /**
+     * Flags to include/exclude certain response data
+     * audioOut - include/exclude response audio buffer (if available)
+     * screenOut - include/exclude response screen output (HTML) (if available)
+     *
+     * @example
+     * ```javascript
+     * const action = new ActionsOnGoogleAva(CREDENTIALS);
+     * action.include.audioOut = true;
+     * action.include.screenOut = true;
+     * action.include = { audioOut: true, screenOut = false };
+     * ```
+     *
+     * @public
+     */
+    include = {
```

Comment:
I was thinking about this new field over the weekend. Does `include` make the most sense as a name? Is there something that would be a better fit?

Reply:
naming things is difficult. when using it like that I think it is meaningful:

    action.include.audioOut = true;